KlipTok.pl.json

```diff
@@ -15,12 +15,49 @@
 
 	// Login View
 	"DarkMode": "Ciemny motyw",
+	"LoginWithTwitch": "Zaloguj się z Twitch",
+	"Logout": "Wyloguj się",
 
+	// Header
+	"SearchForChannel": "Znajdź kanał...",
+	"SearchNotFoundFormat": "Nie znaleziono {0}",
 
 	// Sidebar
-	"SuggestedStreamers": "Proponowani streamerzy",
-	"FollowedStreamers": "Obserwowani streamerzy",
-	"AllStreamers": "Wszyscy streamerzy",
+	"SuggestedStreamers": "Polecane Kanały",
+	"FollowedStreamers": "Obserwowane Kanały",
+	"AllStreamers": "Wszystkie Kanały",
```

Comment:
```suggestion
	"SuggestedStreamers": "Polecani streamerzy",
	"FollowedStreamers": "Obserwowani streamerzy",
	"AllStreamers": "Wszyscy streamerzy",
```

Reply:
When you look at Polish localization on Twitch, they use 'Kanały' (Channels) instead 'Streamerzy' (Streamers).

Reply:
also en-US capitalization

Reply:
Yes, they use, but on KlipTok is 'Streamers' not 'Channels' so I don't think we should change the meaning.